include/zenoh-pico/system/link/raweth.h

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+#ifndef ZENOH_PICO_SYSTEM_LINK_RAWETH_H
+#define ZENOH_PICO_SYSTEM_LINK_RAWETH_H
+
+#include <stdint.h>
+
+#include "zenoh-pico/collections/string.h"
+#include "zenoh-pico/system/platform.h"
+
+#if Z_FEATURE_RAWETH_TRANSPORT == 1
+
+// Ethernet types
+#define _ZP_ETH_TYPE_VLAN 0x8100   // 2048  (0x0800) IPv4
```

Comment:
I believe the `EthType` should be configurable as part of the raweth configuration.

Reply:
Not sure what you mean, the VLAN type is fixed and I doubt we'll need double tagging.

For the frame, we'll set the field as the data length so there is no eth type.

Reply:
So are you suggesting to go with a 802.3 header instead of a Ethernet II header?